engines/bastion_katello/app/assets/javascripts/bastion_katello/organizations/fenced-pages.service.js

```diff
@@ -21,7 +21,11 @@ angular.module('Bastion.organizations').service('FencedPages', ['$state',
             'products',
             'puppet-modules',
             'subscriptions',
-            'sync-plans'
+            'sync-plans',
+            'files',
+            'module-streams',
```

Comment:
I'm not sure this will affect a react page?

Reply:
I don't see how it would